src/extension/options-page/Welcome/index.tsx

```diff
@@ -68,7 +68,7 @@ const WelcomeActions = {
         this.autoVerifyBio(user, prove)
     },
     onFinish(reason: 'quit' | 'done') {
-        window.close()
+        ;((this as unknown) as { props: RouteComponentProps }).props.history.replace('/')
```

Comment:
Why?

Reply:
You can change the parameter of this function.

Reply:
Using `this` is quite common these days... maybe. So how to make typescript recognize `this` the graceful way?